x-pack/platform/plugins/shared/fleet/server/services/epm/archive/parse.ts

```diff
@@ -131,6 +131,7 @@ const optionalArchivePackageProps: readonly OptionalPackageProp[] = [
   'elasticsearch',
   'description',
   'format_version',
+  'discovery',
```

Comment:
nit: worth adding that field to existing tests?

Reply:
added unit test